src/main/g8/default.properties

```diff
@@ -1,5 +1,4 @@
 name = akka-quickstart-java
 description = Akka is a toolkit and runtime for building highly concurrent, distributed, and fault tolerant event-driven apps. This simple application will get you started building Actor based systems with Java.
-scala_version=2.12.8
-# TODO: update to stable once 2.6 has been released
-akka_version=2.6.0-RC1
+scala_version=2.13.1
+akka_version=maven(com.typesafe.akka, akka-actor_2.13, stable)
```

Comment:
shouldn't it be akka-actor-typed?
and there is a reference to _2.12 in pom.xml

Reply:
either should be OK, until we stop publishing `akka-actor_2.13` artifacts ;)

Reply:
I see, this is only for fetching the latest version